### PR TITLE
Update cl_init.lua

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lua/fishing_mod/cl_init.lua [hunk 1]
```diff
@@ -1,7 +1,5 @@
 fishingmod = fishingmod or {}
 
-local color_white = color_white or Color(255, 255, 255, 255)
```
Comment (Zeni44) [inline lua/fishing_mod/cl_init.lua:3]: But why??

**Review by Zeni44 — CHANGES_REQUESTED** [review]

File: lua/fishing_mod/cl_init.lua [hunk 2]
```diff
@@ -51,7 +49,7 @@ function fishingmod.RemoveCatch(name)
 	fishingmod.CatchTable[name] = nil
 end
 
-for key, name in pairs(file.Find("fishing_mod/catch/*.lua", "LUA")) do
+for key, name in ipairs(file.Find("fishing_mod/catch/*.lua", "LUA")) do
```
Comment (Cynosphere) [inline lua/fishing_mod/cl_init.lua:52]: Stub out `key` for `_` as well if you're going to micro-optimize

File: lua/fishing_mod/cl_init.lua [hunk 3]
```diff
@@ -141,7 +139,7 @@ hook.Add("ShouldDrawLocalPlayer", "Fishingmod:ShouldDrawLocalPlayer", function(p
 end)
 
 timer.Create("Fishingmod:Tick", 2, 0, function()
-	for key, entity in pairs(ents.GetAll()) do
+	for key, entity in ipairs(ents.GetAll()) do
```
Comment (Cynosphere) [inline lua/fishing_mod/cl_init.lua:142]: Same as above, stubbing out `key` for `_`

File: lua/fishing_mod/cl_init.lua (hunk 1 above)
Comment (ghost) [reply lua/fishing_mod/cl_init.lua:3]: Micro-optimization

Comment (Zeni44) [reply lua/fishing_mod/cl_init.lua:3]: That is not micro-optimization, you are doing the opposite here. 
Grabbing from a local is faster than global lookup for color_white

Comment (ghost) [reply lua/fishing_mod/cl_init.lua:3]: It's micro-optimization compared to caching. Here you create a variable for nothing.

Comment (Cynosphere) [reply lua/fishing_mod/cl_init.lua:3]: It is used 6 other times thus isn't "for nothing"

**Comment by Earu** [comment]
I'm pretty sure we all agree we're not gonna merge this.

**Pull request closed without merging**